src/main/python/afp_cli/aws_credentials_file.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from six.moves import configparser
+import os
+import six
+
+def write(aws_credentials, filename=None, profile_name='default'):
+
+    if six.PY2:
+        # WTF
+        ORIG_DEFAULTSECT = configparser.DEFAULTSECT
+        configparser.DEFAULTSECT = 'default'
+
+    try:
+
+        if not filename:
+            filename = os.path.expanduser("~") + '/.aws/credentials'
+
+        if not os.path.exists(os.path.dirname(filename)):
+            os.makedirs(os.path.dirname(filename))
+
+        config = configparser.RawConfigParser()
+        config.read(filename)
+
+        if not config.has_section(profile_name) and (profile_name.lower() != 'default' or six.PY3):
+            config.add_section(profile_name)
+
+        config.set(profile_name, 'aws_access_key_id', aws_credentials['AWS_ACCESS_KEY_ID'])
+        config.set(profile_name, 'aws_secret_access_key', aws_credentials['AWS_SECRET_ACCESS_KEY'])
+        config.set(profile_name, 'aws_session_token', aws_credentials['AWS_SESSION_TOKEN'])
```

Comment:
IMHO it would be a good idea to also write the expiration, even if the AWS tools don't read that. But then at least a human could easily notice that the credentials expired. Or will afp-cli remove the credential file (or renew it) when the credentials expire?